android/app/build.gradle

```diff
@@ -189,6 +189,8 @@ dependencies {
 
   // BEGIN_SDK_UNVERSIONED
   implementation(project(path: ':expoview'))
+
+  implementation 'com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava'
```

Comment:
Is that correct? 🤔 
I can read that:
```The idea is: - If users want only ListenableFuture, they depend on listenablefuture-1.0.```
on https://mvnrepository.com/artifact/com.google.guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava
We are not using `guava` so why not to use `listenablefuture:1.0`?

Reply:
There is an error in google libraries which leads to duplicate implementation. This is official fix provided by goggle.

Reply:
Bizarre one though 😂